1.9.0

```diff
@@ -772,6 +772,21 @@
         }
       ]
     },
+    {
+      "sObjectName":"Opportunity",
+      "description": "Update Stage Picklist Values",
+      "taskList":[
+        {
+          "jiraTicket":"ac-64579",
+          "actionType":"PicklistValueStandard",
+          "actionTypeName":"StageName",
+          "optionsMap":{
+            "operation" : "deactivate",
+            "picklistValues":"Negotiation,Develop Proposal,Client Presentation,Initiate Transfer,Discovery"
+          }
+        }
+      ]
+    },
```

Comment:
Need to remove this from this pr right?

Reply:
Youre right, this is not necessary in this pr. Removed.